### ISSUE TITLE
Mangaae cloudflare

### ISSUE THREAD

**Issue by mged** [inorichi/tachiyomi-extensions#2605]
tried on both app versions dev and 0.8.5 not working

the extension is the latest version

first teamx then azora and now mangaae, this app is unusable for arab users now sadlly, it would be nice to fix at least mangaae

**Comment by bbtora** [comment]
What is the error showing, please share a screenshot.

**Comment by mged** [comment]
![Capture2](https://user-images.githubusercontent.com/19857166/78548453-7fa85b00-7809-11ea-8798-74b75ec64175.PNG)

this is the error its a cloudflare error as you can see, right now im unable to read manga on the app due to almost... well all of the good manga soruces has stopped working on this app from long time ago sadly.

**Comment by bbtora** [comment]
Open the source in webview and see if it's opening or not. Then check again in catalog browse.

**Comment by mged** [comment]
same thing not opening, they have protection i guess

it work in webview but when i open it in the app it wont work and show cloudflare issue, please help

**Comment by bbtora** [comment]
It works in webview but doesn't open inside the app? Can you share screenshot of the site opening in webview?

**Comment by mged** [comment]
yes the website works in the webview it even show the chapters 

but in app it does not work and show cloudflare bypass error

**Comment by bbtora** [comment]
Wait for a dev to look into this issue.

**Comment by mged** [comment]
yeah i will wait too, hope they can fix it coz there is NO MORE ARABIC sorucess left to read. all the left ones are either extremly old and outdated or dead sites
